src/test/run-pass/mir_misc_casts.rs

```diff
@@ -120,7 +120,7 @@ fn from_1i8()
     let c10 = 1_i8 as u64;
     let c11 = 1_i8 as f32;
     let c12 = 1_i8 as f64;
-    let c13 = 1_i8 as *const libc::FILE;
+    let c13 = 1_i8 as isize as *const libc::FILE;
```

Comment:
The correct cast btw is through `usize`. Although... I'd leave the tests unchanged and just allow the lint in them.